src/Runner.Worker/Handlers/NodeScriptActionHandler.cs

```diff
@@ -122,9 +122,9 @@ public async Task RunAsync(ActionRunStage stage)
                 else
                 {
                     var exitCode = await step;
+                    Trace.Info($"Node Action run completed with exit code {exitCode}");
```

Comment:
Debug or Info? Info means we need to check runner diag log, Debug goes to user log when ACTIONS_STEP_DEBUG=true

Reply:
Good point. I do think Debug makes more sense there. I went ahead and also updated it for the container actions since the same reasoning should apply there as wel